Gifski/Tooltip.swift

```diff
@@ -1,20 +1,33 @@
 import AppKit
 
 final class Tooltip: NSPopover {
-	init(text: String, closeOnClick: Bool = true, contentInsets: NSEdgeInsets = .init(all: 10.0), maxWidth: Double? = nil) {
+	private let showBehavior: ShowBehavior
+
+	init(text: String, showBehavior: ShowBehavior, closeOnClick: Bool = true, contentInsets: NSEdgeInsets = .init(all: 10.0), maxWidth: Double? = nil) {
```

Comment:
Since the `identifier` is required regardless, wouldn't it be better to just have it as a top-level argument here?

`Tooltip(identifier: "sadad", text: "sdas", showBehavior: .once)`

Reply:
Well, yeah but then I wouldn't be able to use it as a separate type (I need to store the identifier inside the ShowBehavior). I could try to use a struct/class:
```swift
struct ShowBehavior {
	enum Option {
		case always
		case once
	}

	let identifier: String
	let option: Option

	static func always(identifier: String) -> ShowBehavior {
		return .init(identifier: identifier, option: .always)
	}

	static func once(identifier: String) -> ShowBehavior {
		return .init(identifier: identifier, option: .once)
	}
}
``` 

what do you think?

Reply:
👍

Reply:
done 👍